docs/src/examples/components/Embed/Types/EmbedExample.shorthand.tsx

```diff
@@ -6,7 +6,6 @@ const EmbedExample = () => (
     <Embed
       placeholder="https://raw.githubusercontent.com/bower-media-samples/big-buck-bunny-480p-5s/master/poster.jpg"
       video="https://raw.githubusercontent.com/bower-media-samples/big-buck-bunny-1080p-5s/master/video.mp4"
-      defaultActive={true}
```

Comment:
This prop makes example a little bit strange, I expect that will be hidden by default

Reply:
Should we add example for it if we remove it from here?

Reply:
Will add 👍